packages/adblocker/test/engine/engine.test.ts

```diff
@@ -1594,22 +1694,26 @@ function () { console.log(2) }`;
 });
 
 describe('diff updates', () => {
-  function testUpdates(name: string, baseFilters: string[]): void {
+  function testUpdates(name: string, baseFilters: () => string[]): void {
     describe(name, () => {
-      const base = Engine.parse(baseFilters.join('\n'), {
-        debug: false,
-        enableCompression: false,
-        enableOptimizations: false,
-        integrityCheck: false,
-        loadCosmeticFilters: false,
-        loadGenericCosmeticsFilters: false,
-        loadNetworkFilters: true,
-      });
-      const baseSerialized = base.serialize();
-
+      let base: Engine;
```

Comment:
those has been moved from describe top level into `before` so that it is lazily initialised